### PR TITLE
docs(general): SEO optimizations for WPFtoAngular #1447

### PR DESCRIPTION
**Pull request by astaev**
#1447

### CODE REVIEW

File: en/components/general/wpf_to_angular_guide/wpf_to_angular_guide.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
-_keywords: WPF, Windows Presentation Foundation, Angular, Ignite UI for Angular
+title: WPF to Angular tutorial | IgniteUI for Angular | Infragistics
+_description: Learn how to switch from Angular to WPF, by comparing both frameworks. Get familiar with some similarities like binding, events and components.
+_keywords: angular tutorial, igniteui for angular, infragistics
 ---
 
 ## WPF to Angular guide
 
-If you are a WPF developer, this guide will help you to learn Angular by comparing the similarities and differences between the two platforms. The guide is divided into the following topics:
+WPF to Angular tutorial is the first step of a desktop to web framework transition. This tutorial targets WPF developers, who make their first steps to web development. It examines some differences and similarities of both frameworks - applications structure, binding,events, components, etc. The guide is divided into the following topics:
```
Comment (zdrawku) [inline en/components/general/wpf_to_angular_guide/wpf_to_angular_guide.md]:
```suggestion
This WPF to Angular tutorial is the first step that you should take in order to make a smooth transition from a `desktop` to `web` framework. The tutorial targets WPF developers, who make their first steps to web development and it examines some differences and similarities of both frameworks - applications structure, data binding, events, components, etc. The guide is divided into the following topics:
```

Comment (astaev) [reply en/components/general/wpf_to_angular_guide/wpf_to_angular_guide.md]: Suggestions are applied

File: en/components/general/wpf_to_angular_guide/wpf_to_angular_guide.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
 ---
-title: WPF to Angular guide
-_description: WPF to Angular guide topics help you switch from WPF to Angular.
-_keywords: WPF, Windows Presentation Foundation, Angular, Ignite UI for Angular
+title: WPF to Angular tutorial | IgniteUI for Angular | Infragistics
```
Comment (zdrawku) [inline en/components/general/wpf_to_angular_guide/wpf_to_angular_guide.md]:
```suggestion
title: angular tutorial | IgniteUI for Angular | Infragistics
```

Comment (astaev) [reply en/components/general/wpf_to_angular_guide/wpf_to_angular_guide.md]: Forgot them... Everything is updated

**Pull request merged**